extension/js/popup.js

```diff
@@ -0,0 +1,10 @@
+function popup() {
+  document.querySelector("#generate-alias").addEventListener("click", async (e) => {
+    const newRelayAddressResponse = await browser.runtime.sendMessage({
```

Comment:
@bakulf this part doesn't seem to wait for the background script to respond - it goes straight ahead to the `alert` call before it has the `newRelayAddressResponse`

Reply:
This doesn't seem correct in my tests. I receive "undefined" because, in `makeRelayAddress`, I don't have the apiToken. Then, this code runs:
```

  if (!apiToken.apiToken) {
    browser.tabs.create({
      url: RELAY_SITE_ORIGIN,
    });
    return;
  }
```

Reply:
I still have `makeRelayAddress` return early if there's no apiToken, but I added code so that `popup.js` won't try to display the response.